src/util.cc

```diff
@@ -100,11 +102,31 @@ static void MakeUtf8String(Isolate* isolate,
                            MaybeStackBuffer<T>* target) {
   Local<String> string;
   if (!value->ToString(isolate->GetCurrentContext()).ToLocal(&string)) return;
+  String::ValueView value_view(isolate, string);
+
+  auto value_length = value_view.length();
+
+  if (value_view.is_one_byte()) {
+    auto const_char = reinterpret_cast<const char*>(value_view.data8());
+    auto expected_length =
+        simdutf::utf8_length_from_latin1(const_char, value_length);
```

Comment:
```suggestion
    auto expected_length = value_length * 4;
```

Would avoid the `utf8_length_from_latin1` at the cost of more allocation

Reply:
Another option would be 

```
    auto expected_length = value_length > 1024 / 4
       ? simdutf::utf8_length_from_latin1(const_char, value_length);
       : value_length * 4;
```

Reply:
@ronag Hmmm.... I would guess that `value_length * 2` would suffice.

Reply:
Isn't maximum utf8 char length 4 bytes?

Reply:
minimum utf8 length is 2 and this is converting latin1 to utf8 so it will always be 2, I suppose?

Reply:
Updated

Reply:
I've updated the code as well